packages/samples/vuejs/src/components/ResultsList.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import {onMounted} from 'vue';
+import resultTemplate from '../templates/result-template.html?raw';
```

Comment:
I tried sooo many different things, how do you guys feel about this approach?

Reply:
it's the only one that worked, other than what Jean Felix had done before, which is setting the `v-html` of the component to a string - I guess we could do that too to avoid the boilerplate?
```
<atomic-result-list
      fields-to-include="ec_price,ec_rating,ec_images,ec_brand,cat_platform,cat_condition,cat_categories,cat_review_count,cat_color"
      display="grid"
      image-size="large"
      v-html={{resultTemplate}}
    >
    </atomic-result-list>
```

Reply:
So it works if we do this ?:
```
import resultTemplate from '../templates/result-template.html?raw';
[...]
<atomic-result-list v-html={{resultTemplate}} > </atomic-result-list>
```

If so... that does not look too bad ? 🤔 

You have to know the "recipe" of course, but we can document and recommend this approach so that our clients don't have to go through the same process and have to figure it out on their own.

Reply:
I think both are different way of doing the same thing: we can't put the template as JSX, we need to insert it as html. I didn't know about the `?raw`, using `v-html` seemed to be the way to go with Vue when I played with it

Reply:
I like it!